src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.Apps/AppListItem.cs

```diff
@@ -24,25 +24,17 @@ public AppListItem(AppItem app)
         {
             Title = this.Title,
             HeroImage = ((AppCommand)Command!).Icon ?? new IconInfo(string.Empty),
-            Body = "### App",
+            Body = "### " + app.Type,
         };
 
-        if (string.IsNullOrEmpty(app.UserModelId))
+        if (!_app.IsPackaged)
```

Comment:
nit: we can collapse these two branches down since they're the same now

Reply:
hah. of course :)